client/src/pages/course/student/cross-check-review.tsx

```diff
@@ -53,16 +53,21 @@ function Page(props: CoursePageProps) {
   ] = useCriteriaState();
 
   const courseService = useMemo(() => new CourseService(props.course.id), [props.course.id]);
-  const taskServise = new TaskService();
+  const taskService = new TaskService();
 
   const { value: courseTasks = [] } = useAsync(() => courseService.getCourseCrossCheckTasks(), [props.course.id]);
 
   const loadStudentScoreHistory = async (githubId: string) => {
-    const taskCriteriaData = await taskServise.getCriteriaForCourseTask(criteriaId as number);
+    setState({ loading: true, data: [] });
+
+    const [taskCriteriaData, result] = await Promise.all([
+      taskService.getCriteriaForCourseTask(criteriaId as number),
+      courseService.getTaskSolutionResult(githubId, courseTaskId as number),
+    ]);
```

Comment:
We should avoid typecasts as much as possible. It seems that we should have both `taskCriteriaData` and `courseTaskId` to make proper requests, so it's better to have early return if any of these values is `null` instead of typecast here